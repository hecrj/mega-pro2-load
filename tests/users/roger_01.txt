6 
70 100 80 90 50 120
7
2 4 6 0 0 7 0 0 1 5 0 0 3 0 0 
10
2 1 2
20
3 2 3 5
10
2 1 6
40
2 3 4
20
2 3 5
20
1 1
10
2 2 3

-1 1 100

-1 1 101

-2 102

-5 100 101

-3 102

-1 2 200

-1 3 300

-4 4
1 5
0

-1 5 400

-5 100 400

-6
