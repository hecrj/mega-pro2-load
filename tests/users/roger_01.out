Peticion procesada y servidores
0 4
6 
Peticion procesada y servidores
1 4
1 3 
Peticiones pendientes
0 1 100 104
1 1 101 105
Pelicula mas solicitada
1 2
Servidores ocupados
1 1 3
3 1 3
6 0 2
Peticion procesada y servidores
2 4
1 2 
Peticion procesada y servidores
3 2
2 4 7 
Peticion procesada y servidores
4 1
2 4 
Pelicula mas solicitada
1 2

